server/api/field.go

```diff
@@ -36,6 +36,7 @@ type Field struct {
 
 	// Select props
 	SelectRefresh       bool           `json:"refresh,omitempty"`
+	SelectMultiselect   bool           `json:"multiselect,omitempty"`
```

Comment:
@jfrerich 1/5 `SelectMultiselect` is consistent, but redundant a name. Perhaps, just `Multiselect`, or even `Multi`?

@mickmister this (multiple selection support) seems like a bigger deal than just adding the flag for it, or are other pieces in place? Autocomplete worries me in particular.

Reply:
I think the `Select` prefix is valuable because it is grouping Field values that are explicitly related to a select form.  I suppose a `SelectIsMulti` my be more intention revealing, implying a boolean option for a select field

Reply:
For reference, I have not tested this for autocompletion, but have multiselect fields working in the zendesk app in this PR https://github.com/mattermost/mattermost-app-zendesk/pull/21

Reply:
> but have multiselect fields working in the zendesk app in this PR

Are you saying that it is already supported in the webapp?

Reply:
Yes. There is an oddity where the multiselect opens with an empty value already in the select box, but it is working.

Reply:
![image](https://user-images.githubusercontent.com/7575921/105883591-a6b85f00-5fcc-11eb-85d5-189d221406d9.png)

Reply:
Here is the multiselect with multiple selected fields

![image](https://user-images.githubusercontent.com/7575921/105883656-b46de480-5fcc-11eb-9b56-8befc123c9f1.png)

Reply:
> Yes. There is an oddity where the multiselect opens with an empty value already in the select box, but it is working.

Can you make a ticket for this if it is not resolved?

Reply:
1/5 `SelectIsMulti` sounds good to me